.mailmap

```diff
@@ -56,21 +137,83 @@ Nathan Woodrow <madmanwoo@gmail.com> <nathan_woodrow@technologyonecorp.com>
 Nathan Woodrow <madmanwoo@gmail.com> <woodrow.nathan@gmail.com>
 Nathan Woodrow <madmanwoo@gmail.com> <woo@woo-mint.(none)>
 
+Nedjima Belgacem <gb_nedjima@esi.dz> NEDJIMAbelgacem <gb_nedjima@esi.dz>
+Nedjima Belgacem <gb_nedjima@esi.dz> nedjima <gb_nedjima@esi.dz>
+Nedjima Belgacem <gb_nedjima@esi.dz> Belgacem <gb_nedjima@esi.dz>
+
+Nicolas Godet <nicolas.godet@outlook.fr> nicogodet <39594821+nicogodet@users.noreply.github.com>
+Nicolas Godet <nicolas.godet@outlook.fr> nicogodet <nicolas.godet@outlook.fr>
+Nicolas Godet <nicolas.godet@outlook.fr> <39594821+nicogodet@users.noreply.github.com>
+
+Nyall Dawson <nyall.dawson@gmail.com> nyalldawson <nyall.dawson@gmail.com>
+Nyall Dawson <nyall.dawson@gmail.com> <nyall@localhost.localdomain>
+Nyall Dawson <nyall.dawson@gmail.com> Nyall <nyall.dawson@gmail.com>
+
+Olivier Dalang <olivier.dalang@gmail.com> olivierdalang <olivier.dalang@epfl.ch>
+Olivier Dalang <olivier.dalang@gmail.com> olivierdalang <olivier.dalang@gmail.com>
+Olivier Dalang <olivier.dalang@gmail.com> olivier <olivier.dalang@gmail.com>
+
+Otto Dassau <dassau@gbd-consult.de> dassau <dassau@c8812cc2-4d05-0410-92ff-de0c093fc19c>
+Otto Dassau <dassau@gbd-consult.de> dassau <dassau@gbd-consult.de>
+
 Paolo Cavallini <cavallini@faunalia.it>
 Paolo Cavallini <cavallini@faunalia.it> <pcav@c8812cc2-4d05-0410-92ff-de0c093fc19c>
 
+Paul Blottiere <blottiere.paul@gmail.com> Blottiere Paul <blottiere.paul@gmail.com>
+
+Peter Petrik <zilolv@gmail.com> PeterPetrik <zilolv@gmail.com>
+
 Radim Blazek <radim.blazek@gmail.com> <rblazek@c8812cc2-4d05-0410-92ff-de0c093fc19c>
 
+Raymond Nijssen <r.nijssen@terglobo.nl> raymondnijssen <r.nijssen@terglobo.nl>
+
+Régis Haubourg <regis.haubourg@eau-adour-garonne.fr> Régis Haubourg <regis.haubourg@oslandia.com>
```

Comment:
```suggestion
Régis Haubourg <regis@qgis.org> 
```

Reply:
The two e-mail accounts included in the mapping are found in the commit history. That's why they should appear in the file. 

They are now mapped to <regis@qgis.org> in 6c9a82c